app/src/components/v-form/v-form.vue

```diff
@@ -222,6 +222,23 @@ export default defineComponent({
 				throw new Error('[v-form]: You need to pass either the collection or fields prop.');
 			});
 
+			const defaultValues = computed(() => {
+				return fields.value.reduce(function (acc, field) {
+					if (
+						field.schema?.default_value !== undefined &&
+						// Ignore autoincremented integer PK field
+						!(
+							field.schema.is_primary_key &&
+							field.schema.data_type === 'integer' &&
+							typeof field.schema.default_value === 'string'
+						)
```

Comment:
Have you considered functions like `now()` as default values?

Reply:
Nope, I did not. Just tested with a timestamp field with auto modified time (on create & update) and there was no issue. 
Is there any interface that I should take note of?

Reply:
It is not specific to the `interface` but to `default value`.
If you choose, for example, type `string` and on `default value` you could put `concat('Direcuts', 'is awesome')`.
Maybe you can split `field.schema.default_value` by `(` and select first position? Not sure if it is the proper way.

Reply:
My mistake.
Thank you for notice me and sorry for causing some extra noise. I thought we supported functions as default values but it seems only `now()` is supported which is transformed into `CURRENT_TIMESTAMP`.

That being said, looks great to me 😉